### PR TITLE
adding test for incrementing network config ids

### PR DESCRIPTION
**Pull request by galenmarchetti**

### CODE REVIEW

File: commons/networks/service_network_config_test.go [hunk 1]
```diff
@@ -76,6 +76,33 @@ func TestDisallowingNonexistentDependencies(t *testing.T) {
 
 // TODO test configuration IDs get incremented!
 
+func TestNetworkIdsDifferent(t *testing.T) {
```
Comment (mieubrisse) [inline commons/networks/service_network_config_test.go]: Thsi should be `TestConfigurationIdsDifferent`

File: commons/networks/service_network_config_test.go [hunk 2]
```diff
@@ -76,6 +76,33 @@ func TestDisallowingNonexistentDependencies(t *testing.T) {
...
+	testImage := "testImage"
+	expectedOrder := []int{
```
Comment (mieubrisse) [inline commons/networks/service_network_config_test.go]: So this actually makes an assertion about the IDs that will come back (i.e. that they'll be sequentially-increasing ints starting at 0), which implicitly means this test knows about how the internals of the function work. IMO that should be abstracted and we should instead just assert that they're different - make a set-of-IDs-returned and assert that several calls to both `AddStaticImageConfiguration` and `AddTestImageConfiguration` don't collide with anything that already exists

**Review by mieubrisse — APPROVED** [review]
Approved pending comment addressal

File: commons/networks/service_network_config_test.go (hunk 1 above)
Comment (galenmarchetti) [reply commons/networks/service_network_config_test.go]: done

File: commons/networks/service_network_config_test.go (hunk 2 above)
Comment (galenmarchetti) [reply commons/networks/service_network_config_test.go]: So the goal of this test is: "Add test to ensure that configuration IDs are incremented in the call to SvcNetworkCfgBuilder.AddConfiguration". Do we not want to test that they're incremented, but rather just that they're different?

Comment (galenmarchetti) [reply commons/networks/service_network_config_test.go]: and changed regardless

**Pull request merged**